src/core_plugins/kibana/public/visualize/editor/sidebar.html

```diff
@@ -14,31 +16,53 @@
         <!-- tabs -->
         <ul class="nav navbar-nav">
           <li ng-class="{active: sidebar.section == 'data'}" ng-show="sidebar.showData">
-            <a class="navbar-link active" ng-click="sidebar.section='data'">Data</a>
+            <a
+              class="vis-editor-subnav-link"
+              g-class="{'is-vis-editor-sub-nav-link-selected': sidebar.section == 'data'}"
+              g-click="sidebar.section='data'"
+            >Data</a>
```

Comment:
I think this will be more maintainable if we separate the opening and closing tags from the content:

```html
>
  Data
</a>
```

I think this consistency will let us change the content, e.g. adding more text, adding new elements, without having to add and remove new lines  depending on how much content there is. What do you think?